r/R/dplyr-functions.R

```diff
@@ -848,6 +848,21 @@ nse_funcs$wday <- function(x,
   Expression$create("day_of_week", x, options = list(count_from_zero = FALSE, week_start = week_start))
 }
 
+nse_funcs$is.Date <- function(x) {
+  lubridate::is.Date(x) ||
```

Comment:
lubridate isn't in the arrow imports or depends so I don't think we can call lubridate functions it like this. I think we need to use `class()` or `inherits()` to determine if `x` is a lubridate object when `x` is an R object not an arrow object.

Reply:
Unlikely, but it shouldn't fail if you don't have lubridate installed

Reply:
We don't have this in our CI yet, but would be a test case for https://issues.apache.org/jira/browse/ARROW-14544